<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenMenu</class>
 <widget class="QDialog" name="OpenMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qlink</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/open.jpg);</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>载入游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>怎么玩？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>没有可供载入的存档哦！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
